src/platforms/kotlin-multiplatform/dsym.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Upload Debug Symbols
+sidebar_order: 1
+description: "Learn more about how to upload debug symbols in Sentry Kotlin Multiplatform."
+---
+
+To symbolicate your stacktraces, you need to provide debug information to Sentry.
+The symbolication process unscrambles the stacktraces to reveal the function, file names, and line numbers of the crash.
+
+## Apple
+
+For Apple applications please follow the iOS documentation on [Uploading Debug Symbols](/platforms/apple/guides/ios/dsym) to set up debug symbols upload.
+
+## Android
+
+### Gradle
+
+To upload ProGuard mapping files via Gradle you need to install the [Sentry Android Gradle](/platforms/android/gradle) plugin:
+
+```kotlin {filename:androidApp/build.gradle.kts}
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id("com.android.application")
+    id("io.sentry.android.gradle") version "{{@inject packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
```

Comment:
I'd add either a code snippet to the plugin's feature `uploadNativeSymbols` or at least mention it.